packages/dd-trace/src/plugins/database.js

```diff
@@ -4,6 +4,61 @@ const StoragePlugin = require('./storage')
 
 class DatabasePlugin extends StoragePlugin {
   static get operation () { return 'query' }
+
+  constructor (...args) {
+    super(...args)
+    this.serviceTags = {
+      dddbs: '',
+      encodedDddbs: '',
+      dde: '',
+      encodedDde: '',
+      ddps: '',
+      encodedDdps: '',
+      ddpv: '',
+      encodedDdpv: ''
+    }
+  }
+  propagationHelper (serviceTag, encodeATag, spanConfig) {
+    if (serviceTag !== spanConfig) {
+      this.serviceTags[serviceTag] = spanConfig
+      this.serviceTags[encodeATag] = encodeURIComponent(spanConfig)
+    }
+  }
+
+  createDBMPropagationCommentService () {
+    const local = this.serviceTags
```

Comment:
This variable name is not very descriptive. However, it's actually not even needed and instead individual variables could be used with for example `const { encodedDddbd  } = this.serviceTags`.

Reply:
Very true, updated to `const { encodedDddbs, encodedDde, encodedDdps, encodedDdpv } = this.serviceTags` which is now called after using the `encodingServiceTags` helper